lib/form1010_ezr/service.rb

```diff
@@ -29,21 +29,23 @@ def initialize(user)
     def submit_async(parsed_form)
       HCA::EzrSubmissionJob.perform_async(
         HealthCareApplication::LOCKBOX.encrypt(parsed_form.to_json),
-        HealthCareApplication.get_user_identifier(@user)
+        @user.uuid
       )
 
       { success: true, formSubmissionId: nil, timestamp: nil }
     end
 
     def submit_sync(parsed_form)
       res = with_monitoring do
-        es_submit(parsed_form, @user, FORM_ID)
+        es_submit(parsed_form, HealthCareApplication.get_user_identifier(@user), FORM_ID)
```

Comment:
The main reason is because in the `es_submit` method, we're calling `HCA::EnrollmentSystem.veteran_to_save_submit_form`, which needs the user identifier hash. It looks like this:
```
def self.get_user_identifier(user)
  return if user.nil?

  {
    'icn' => user.icn,
    'edipi' => user.edipi
  }
end
```